packages/main/src/Tree.hbs

```diff
@@ -4,6 +4,7 @@
     .footerText="{{footerText}}"
     .noDataText="{{noDataText}}"
     .accessibleRole="{{_role}}"
+    .accessibleName="{{_label}}"
```

Comment:
does it make a difference in the render process if I use `accessible-name` here instead?

Reply:
I think this makes no difference for the renderer. 
@ilhan007 could you please confirm this?